### PR TITLE
Add otpauth w/ git auto-update

### PR DESCRIPTION
**Pull request by hectorm**

### CODE REVIEW

**Comment by hectorm** [comment]
Although the project on GitHub has less than 200 stars, it currently receives 34k downloads per month in [npm](https://www.npmjs.com/package/otpauth). But I prefer to add it as a git repository instead of npm. If this is a problem, I can change it to npm.

File: packages/o/otpauth.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "source": "git",
+    "target": "git://github.com/hectorm/otpauth.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "dist/*",
+          "types/*"
+        ]
+      }
```
Comment (MattIPv4) [inline packages/o/otpauth.json:45]:
```suggestion
      {
        "basePath": "dist",
        "files": [
          "*.js?(.map)"
        ]
      },
      {
        "basePath": "types",
        "files": [
          "*.d.ts"
        ]
      }
```
Let's pull the files out of their subdirectories to the root directory when they're on cdnjs, for simplicity.

File: packages/o/otpauth.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      "url": "https://hector.molinero.dev"
+    }
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/hectorm/otpauth",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hectorm/otpauth.git"
+  },
+  "filename": "dist/otpauth.cjs.min.js",
```
Comment (MattIPv4) [inline packages/o/otpauth.json:34]:
```suggestion
  "filename": "otpauth.cjs.min.js",
```
Per other suggestion, let's pull this out to the root.

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

**Comment by hectorm** [comment]
I've commited the suggestions.

**Review by MattIPv4 — APPROVED** [review]
Lgtm, thank you!

**Pull request merged**